### PR TITLE
Added homework 4

### PR DESCRIPTION
**Pull request by Lilly7777**

### CODE REVIEW

File: .github/workflows/homework_04_11_liliya_lyubenova.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      - name: Run unit tests
+        run: |
+          cd homework_04/11_liliya_lyubenova
+          pytest
+        
+      - name: Run Pylint
+        run: |
+          cd homework_04/11_liliya_lyubenova
+          pylint problem.py
+          pylint test_problem.py
```
Comment (mvvrachev) [inline .github/workflows/homework_04_11_liliya_lyubenova.yaml:44]: Everything seems fine.

File: homework_04/11_liliya_lyubenova/test_problem.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+"""Tests for problem.py"""
+from problem import Point, Circle, RelativePosition
+
+def test_no_common_points():
+    """First test"""
```
Comment (mvvrachev) [inline homework_04/11_liliya_lyubenova/test_problem.py:5]: You might briefly explain what you are testing in the function description but in the current case the name of the test says it all.

File: homework_04/11_liliya_lyubenova/problem.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
+"""Find relative position for 2 circles"""
```
Comment (mvvrachev) [inline homework_04/11_liliya_lyubenova/problem.py:1]: I personally prefer the quotes to be on different lines for better readability but this way is also fine.

**Review by mvvrachev — APPROVED** [review]

File: .github/workflows/homework_04_11_liliya_lyubenova.yaml (hunk 1 above)
Comment (petardmnv) [reply .github/workflows/homework_04_11_liliya_lyubenova.yaml:44]: OK

File: homework_04/11_liliya_lyubenova/problem.py (hunk 3 above)
Comment (Lilly7777) [reply homework_04/11_liliya_lyubenova/problem.py:1]: Thanks, you are right. I will correct them.

**Review by petardmnv — APPROVED** [review]
OK

**Pull request merged**